packages/cli/src/commands/setup/ui/libraries/windicss.js

```diff
@@ -8,10 +8,24 @@ import { errorTelemetry } from '@redwoodjs/telemetry'
 
 import { getPaths } from '../../../../lib'
 import c from '../../../../lib/colors'
+import { printSetupNotes } from '../helpers'
```

Comment:
Just tried running this locally and it seems like this file doesn't exist at this location; this specific helper is co-located with the deploy commands at the moment. Don't see why we can't move it, we probably just never had a reason to

Reply:
Fixed in https://github.com/redwoodjs/redwood/pull/8597/commits/d80b8dfd8240de56eee584c4172e6a8b79e02693